### DISCUSSION TITLE
When we can get Support for Prisma 2.28 in type-graphql ?

### DISCUSSION THREAD

**Discussion by muhad-bk** [MichalLytek/typegraphql-prisma#151] [General]
Hi

Prisma 2.28 solves MongoDB relational query issue and more, So When we can get Support for Prisma 2.28 in typegraphql-prisma?

**Comment by MichalLytek** [comment]
When it's ready 😉

**Reply by muhad-bk** [reply]
Created a PR for updating Prisma version

**Reply by MichalLytek** [reply]
That was not needed. Your PR only bumped deps, you can do that by yourself and skip prisma version check:
https://prisma.typegraphql.com/docs/advanced/prisma-version
